virtwho/register.py

```diff
@@ -271,41 +291,52 @@ def attr_analyzer(self, attr):
             attr_data[key] = value
         return attr_data
 
-    def facts_create(self, key, value):
+    def facts_create(self, key, value, wait=10):
```

Comment:
add a new param `wait`, because the rhsm doesn't need much wait time, but satellite may need about 60s to fix the conflict issue with tasks.